gfx/layers/opengl/GLManager.cpp

```diff
@@ -91,10 +94,14 @@ GLManager::CreateGLManager(LayerManagerComposite* aManager)
 /* static */ GLManager*
 GLManager::CreateGLManager(WebRenderLayerManager* aManager)
```

Comment:
This function signature uses WebRenderLayerManager* outside of the ifdef. Doesn't that fail compilation if webrender is turned off?